docs/build/_index.md

```diff
@@ -1,9 +1,164 @@
+        </div>
+    </div>
+    <div class="col sectionlist">
+<div><h2>Services:</h2><ul class="sectionlist"><li><a href="/data/" title="Data Management Service"><div><picture><img src="/services/icons/data-capture.svg" width="/services/icons/data-capture" height="/services/icons/data-capture" alt="Data Management" loading="lazy"></picture><p>Data Management</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/motion/" title="Motion Service"><div><picture><img src="/services/icons/motion.svg" width="/services/icons/motion" height="/services/icons/motion" alt="Motion" loading="lazy"></picture><p>Motion</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/frame-system/" title="Robot Frame System"><div><picture><img src="/services/icons/frame-system.svg" width="/services/icons/frame-system" height="/services/icons/frame-system" alt="Frame System" loading="lazy"></picture><p>Frame System</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/base-rc/" title="Base Remote Control Service"><div><picture><img src="/services/icons/base-rc.svg" width="/services/icons/base-rc" height="/services/icons/base-rc" alt="Base Remote Control" loading="lazy"></picture><p>Base Remote Control</p></div></a></li></ul><ul class="sectionlist"><li><a href="/ml/" title="ML Model Service"><div><picture><img src="/services/icons/ml.svg" width="/services/icons/ml" height="/services/icons/ml" alt="ML Model" loading="lazy"></picture><p>ML Model</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/navigation/" title="Navigation Service"><div><picture><img src="/services/icons/navigation.svg" width="/services/icons/navigation" height="/services/icons/navigation" alt="Navigation" loading="lazy"></picture><p>Navigation</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/slam/" title="SLAM Service"><div><picture><img src="/services/icons/slam.svg" width="/services/icons/slam" height="/services/icons/slam" alt="SLAM" loading="lazy"></picture><p>SLAM</p></div></a></li></ul><ul class="sectionlist"><li><a href="/ml/vision/" title="Vision Service"><div><picture><img src="/services/icons/vision.svg" width="/services/icons/vision" height="/services/icons/vision" alt="Vision" loading="lazy"></picture><p>Vision</p></div></a></li></ul></div>
+    </div>
+  </div>
+</div>
+
+For additional flexibility see the [modular resource registry](/registry/).
+
+## SDKs
+
+Each category of resource has a standardized API that you can access with an SDK (software development kit) in your preferred programming language.
+For example, you can send the same commands to any kind of motor, using any of the following programming languages:
+
+{{<sectionlist section="/sdks">}}
+
+## Start building
+
+To get started, first [install `viam-server`](/get-started/installation/) on your smart machine's computer.
+Viam supports a wide variety of popular systems, including:
+
+<div id="board-carousel" class="carousel max-page">
```

Comment:
I like the Platform and the Build pages overall - I am concerned with the placement of the carousel and am wondering if we should move this up on the page. Start with that, then the components and services, and then programming. I think you can probably do that and then have the two cards still under a Next Steps header at the bottom - what do you think?

Reply:
Here, I wanted to emphasize the "building blocks" concept before getting into specific hardware, and before getting into the specific "board" component.

Reply:
Okay, and also leave it here? Or move it out of here?

Reply:
Move out of here there - you did say it didn't fit here right?

Reply:
I mean that I think it fits at the bottom of build but not at the top of build.